spyder_line_profiler/tests/test_lineprofiler.py

```diff
@@ -17,7 +17,8 @@
 MAIN_APP = qapplication() 
 
 # Local imports
-from spyder_line_profiler.widgets.lineprofiler import LineProfilerWidget
+from spyder_line_profiler.spyder.widgets import SpyderLineProfilerWidget
+
 
 try:
     from unittest.mock import Mock
```

Comment:
Checking this file maybe is worthy also to remove this try-except import logic since it is doing just a fallback for Python 2